src/app/modules/customer-management/store/customer-management.reducers.ts

```diff
@@ -1,22 +1,26 @@
 import { CustomerManagementActions, CustomerManagementActionTypes } from './customer-management.actions';
-import { Customer } from '../../shared/models/customer.model';
+import { Customer } from 'src/app/modules/shared/models';
 
 export interface CustomerState {
   data: Customer[];
   isLoading: boolean;
   message: string;
+  customerIdToEdit: string;
 }
 
 export const initialState: CustomerState = {
   data: [],
   isLoading: false,
   message: '',
+  customerIdToEdit: '',
 };
 
 export function customerManagementReducer(
   state: CustomerState = initialState,
   action: CustomerManagementActions
 ): CustomerState {
+  const customersList = [...state.data];
+  const key = 'id';
```

Comment:
Please, let's apply the previous comment here too

Reply:
done